app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Api::V1::OrganizationsController < Api::V1::ApiController
+
+  def show
+    render json: resource
+  end
+
+  def index
+    render json: collection
+  end
+
+  def create
+    @organization = Organization.new(organization_params)
+
+    if @organization.save
+      render json: organization
+    else
+      render_error(note.errors.full_messages[0], :unprocessable_entity)
+    end
+  end
+
+  def update
+    @organization = resource
+    if @organization.update(organization_params)
+      render json @organization
+    else
+      render_error @organization.errors.full_messages[0], :unprocessable_entity
+    end
+  end
+
+  def destroy
+    resource.destroy
+    head :no_content
+  end
+
+  def organization_params
+    params.require(:organization).permit(:name, :logo, :logo_cache, :remove_logo)
+  end
+
+  def collection
+    Organization.all
+  end
+
+  def resource
+    collection.where(token: request.headers['token'])
```

Comment:
what's this?
follow up the `Devise` way, make it `current_organization` if u're authorizing an organization via API (weirdo though, should be user i think)